### PR TITLE
[Ordered Comments] Flush to the spec

### PR DESCRIPTION
**Pull request by nex3**
See #2848

### CODE REVIEW

File: spec/at-rules/extend.md [hunk 1]
```diff
@@ -212,15 +218,17 @@ that includes CSS for *all* modules transitively used or forwarded by
...
+    > subsequence is empty and `initial-imports` is inserted at the beginning of
+    > `css`.
 
   * For each top-level statement `statement` in `domestic`'s CSS tree after
     `initial-imports`:
 
-    * If `statement` is an `@import` rule, insert a copy of `statement` in `css`
-      after the last `@import` rule, or at the beginning of `css` if it doesn't
-      contain any `@import` rules.
+    * If `statement` is a marked comment, ignore it.
```
Comment (Goodwine) [inline spec/at-rules/extend.md:231]: I don't see where `statement` gets marked, I think "them mark it" is missing in the bullet point below 🤔

**Review by Goodwine — CHANGES_REQUESTED** [review]

File: spec/at-rules/extend.md (hunk 1 above)
Comment (nex3) [reply spec/at-rules/extend.md:231]: It's marked on line 210, above.

Comment (Goodwine) [reply spec/at-rules/extend.md:231]: That bullet marks a "comment" rather than a `statement`

Comment (nex3) [reply spec/at-rules/extend.md:231]: Right—this bullet checks if `statement` is a comment that is marked.

Comment (Goodwine) [reply spec/at-rules/extend.md:231]: 🤦🏽‍♂️ for some reason I read this as "If `statement` is marked, ignore it" lol

**Review by Goodwine — APPROVED** [review]

**Pull request merged**